test/CSModule.t.sol

```diff
@@ -7224,6 +7224,424 @@ contract CSMDepositableValidatorsCount is CSMCommon {
+        );
+    }
+
+    function test_depositedOnly_UpdateToWorseCurve() public {
+        uint256 noId = createNodeOperator(7);
+        csm.obtainDepositData(7, "");
+
+        (, uint256 requiredBefore) = accounting.getBondSummary(noId);
+        updateToWorseCurve();
+        (, uint256 requiredAfter) = accounting.getBondSummary(noId);
+
+        assertGt(
+            requiredAfter,
+            requiredBefore,
+            "Required bond should increase"
+        );
+        assertEq(
+            csm.getNodeOperator(noId).depositableValidatorsCount,
+            0,
+            "Depositables should not change after curve update"
+        );
+        assertEq(
+            accounting.getUnbondedKeysCountToEject(noId),
+            2,
+            "Should be unbonded keys"
+        );
+
+        csm.normalizeQueue(noId);
+        assertEq(
+            csm.getNodeOperator(noId).depositableValidatorsCount,
+            0,
+            "Depositables should not change after normalization"
+        );
+    }
+
+    function test_depositableOnly_UpdateToBetterCurve() public {
+        uint256 noId = createNodeOperator(7);
+        uint256 depositableBefore = csm
+            .getNodeOperator(noId)
+            .depositableValidatorsCount;
+
+        (, uint256 requiredBefore) = accounting.getBondSummary(noId);
+        updateToBetterCurve();
+        (, uint256 requiredAfter) = accounting.getBondSummary(noId);
+
+        assertGt(
+            requiredBefore,
+            requiredAfter,
+            "Required bond should decrease"
+        );
+        assertEq(
+            csm.getNodeOperator(noId).depositableValidatorsCount,
+            depositableBefore,
+            "Depositables should not change after curve update"
+        );
+        assertEq(
+            accounting.getUnbondedKeysCountToEject(noId),
+            0,
+            "Should be no unbonded keys"
+        );
+
+        csm.normalizeQueue(noId);
+        assertEq(
+            csm.getNodeOperator(noId).depositableValidatorsCount,
+            depositableBefore,
+            "Depositables should not change after normalization"
+        );
+    }
+
+    function test_depositableOnly_UpdateToWorseCurve() public {
+        uint256 noId = createNodeOperator(7);
+        uint256 depositableBefore = csm
+            .getNodeOperator(noId)
+            .depositableValidatorsCount;
+
+        (, uint256 requiredBefore) = accounting.getBondSummary(noId);
+        updateToWorseCurve();
+        (, uint256 requiredAfter) = accounting.getBondSummary(noId);
+
+        assertGt(
+            requiredAfter,
+            requiredBefore,
+            "Required bond should increase"
+        );
+        assertEq(
+            csm.getNodeOperator(noId).depositableValidatorsCount,
+            depositableBefore,
+            "Depositables should not change after curve update"
+        );
+        assertEq(
+            accounting.getUnbondedKeysCountToEject(noId),
```

Comment:
there is no lock, so i'd expect more straightforward `getUnbondedKeysCount` here

Reply:
done